imports/client/components/styling/FixedLayout.tsx

```diff
@@ -3,8 +3,9 @@ import { NavBarHeight } from './constants';
 
 export default styled.div`
   position: fixed;
-  top: calc(env(safe-area-inset-top, 0) + ${NavBarHeight});
-  bottom: 0;
-  left: env(safe-area-inset-left, 0);
-  right: env(safe-area-inset-right, 0);
+  inset: /* top right bottom left */
```

Comment:
Thank you for this comment; I will probably never manage to remember the ordering